data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -21,97 +21,8 @@ MACHINE_CONFIG_CONTROLLER_IMAGE=$(podman run --rm ${release} image machine-confi
 MACHINE_CONFIG_SERVER_IMAGE=$(podman run --rm ${release} image machine-config-server)
 MACHINE_CONFIG_DAEMON_IMAGE=$(podman run --rm ${release} image machine-config-daemon)
 
-KUBE_APISERVER_OPERATOR_IMAGE=$(podman run --rm ${release} image cluster-kube-apiserver-operator)
-KUBE_CONTROLLER_MANAGER_OPERATOR_IMAGE=$(podman run --rm ${release} image cluster-kube-controller-manager-operator)
-KUBE_SCHEDULER_OPERATOR_IMAGE=$(podman run --rm ${release} image cluster-kube-scheduler-operator)
-
-OPENSHIFT_HYPERSHIFT_IMAGE=$(podman run --rm ${release} image hypershift)
-OPENSHIFT_HYPERKUBE_IMAGE=$(podman run --rm ${release} image hyperkube)
-
-CLUSTER_BOOTSTRAP_IMAGE=$(podman run --rm ${release} image cluster-bootstrap)
-
-mkdir --parents ./{bootstrap-manifests,manifests}
-
-if [ ! -d cvo-bootstrap ]
```

Comment:
are you sure you want to include `cvo` in `cluster-bootstrap script` ?

Reply:
It's part of the pivot from bootstrap static pods to the real masters (in other words: bootkube start was waiting for it). If we find a way to exclude it from that step, we can move it back here.